PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,55 @@
+⚛👋 Hello there! Welcome. Please follow the steps below to tell us about your
+contribution.
+
+1. Replace this text with the contents of the template.
+2. Fill in all sections of the template.
+3. Click "Create pull request".
+
+# Pull Request Template
+
+### :octocat: Are you working in some issue? Identify the issue!
+
+Link the issue describing what you're fixing.
+
+If there is not yet an issue, please open a new one and then link it to in your
+pull request. Note: In some cases, one person's "bug" is another person's
+"feature." If the pull request does not address an existing issue with the
+appropriated label (e.g. bug label). Note: the maintainers have the final say
+on whether the current behavior is a bug or not.
```

Comment:
> If the pull request does not address an existing issue with the appropriated label (e.g. bug label)

...then what?

Reply:
The complete phrase is "If the pull request does not address an existing issue with the "bug" label, the maintainers have the final say on whether the current behavior is a bug." I'll remove the incomplete phrase because the Note in this paragraph summarizes the missing idea.